src/guides/bioinformatics/index.rst

```diff
@@ -12,6 +12,7 @@ How-to guides for performing bioinformatic anaylses with Nextstrain.
 
    defining-clades
    augur_snakemake
+   missing_sequence_data
```

Comment:
_non-blocking_

While there's a mix of hyphens and underscores in the pages of this section, I think generally we've tried to standardize page names (and thus URLs) to use hyphens.

Reply:
No problem -- changed in [1a77151](https://github.com/nextstrain/docs.nextstrain.org/pull/158/commits/1a771514d080d35149fed8160839f4f9fbff158e). I tried to follow the established pattern as 6/8 of the files in that folder used underscores but I guess we're consistently inconsistent.

Reply:
Nod, the reason is that those files all come from other repos, via the (still ill-advised, I think) `fetch-docs.py`:

https://github.com/nextstrain/docs.nextstrain.org/blob/27e5f6d921f0c99ce974cbc76046f5d23be1a446/src/fetch-docs.py#L16-L24

Most of the files in this repo itself use hyphens:

```
$ git ls-files src/ | grep -
src/assets/narrative-tutorial-p1.png
src/assets/narrative-tutorial-p2.png
src/fetch-docs.py
src/guides/bioinformatics/defining-clades.md
src/guides/manage-installation.rst
src/guides/share/community-builds.md
src/guides/share/download-data.md
src/guides/share/fetch-via-urls.md
src/guides/share/groups/migrate-from-s3.rst
src/guides/share/sars-cov-2.md
src/images/auspice-components-diversity-panel.png
src/images/auspice-components.png
src/images/branch-hover.png
src/images/diversity-plot.png
src/images/labelled-deme.png
src/images/labelled-map.png
src/images/labelled-movement.png
src/images/local-vs-container.svg
src/learn/augur-to-auspice.rst
src/learn/interpret/how-to-read-a-tree.md
src/learn/interpret/interacting-with-nextstrain.md
src/learn/interpret/map-interpretation.md
src/reference/data-files.rst
src/reference/data-formats.rst
src/snippets/ambient-setup.rst
src/snippets/nextstrain-setup-conda.rst
src/snippets/wsl-home-dir.rst
src/tutorials/creating-a-bacterial-pathogen-workflow.rst
src/tutorials/creating-a-workflow.rst
src/tutorials/narratives-how-to-write.rst
src/tutorials/running-a-workflow.rst

$ git ls-files src/ | grep _
src/images/hidden_toc.png
src/images/infection_tree_combined.png
src/images/narratives_debugger_screenshot.jpg
src/images/submodule_eg.png
src/images/toy_alignment_tree.png
src/images/zika_example.png
```

Reply:
Despite all the work, our docs still feel like a maze. It really would be nice to unify them (as far as the viewer is concerned, behind the scenes we want them to stay in separate repos). But this was pretty technically difficult as I remember, and doesn't seem like something we're going to get time to do anytime soon.

Reply:
I get it, for sure. OTOH though, I have heard independently from a few folks that they really appreciate how well-documented Nextstrain is. Even when they've had trouble finding something, they "know it must be there" and usually can find the info they're looking for. So as much as we tend to see a big todo list for the docs and find them more of a maze than we'd like, it's nice to remember that they've still come a long way and are leaps and bounds better than much software.